src/Lykke.Service.HFT/Lykke.Service.HFT.csproj

```diff
@@ -26,31 +26,22 @@
 
   <ItemGroup>
     <PackageReference Include="AspNetCoreRateLimit" Version="2.1.0" />
-    <PackageReference Include="Lykke.Common" Version="6.6.0" />
-    <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.7.0" />
+    <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.7.1" />
     <PackageReference Include="Lykke.Cqrs" Version="4.5.0" />
     <PackageReference Include="Lykke.Http" Version="1.0.0" />
-    <PackageReference Include="Lykke.Logs" Version="4.3.0" />
-    <PackageReference Include="Lykke.MatchingEngineConnector" Version="4.2.2" />
-    <PackageReference Include="Lykke.Messaging" Version="5.0.0" />
+    <PackageReference Include="Lykke.Logs" Version="4.3.3" />
+    <PackageReference Include="Lykke.MatchingEngineConnector" Version="6.0.2" />
     <PackageReference Include="Lykke.Messaging.RabbitMq" Version="2.0.0" />
-    <PackageReference Include="Lykke.MonitoringServiceApiCaller" Version="1.2.2" />
+    <PackageReference Include="Lykke.MonitoringServiceApiCaller" Version="1.3.0" />
     <PackageReference Include="Lykke.Service.Balances.Client" Version="1.1.0" />
     <PackageReference Include="Lykke.Service.FeeCalculator.Client" Version="2.1.3" />
     <PackageReference Include="Lykke.Service.OperationsHistory.Client" Version="2.1.0" />
     <PackageReference Include="Lykke.Service.OperationsRepository.Contract" Version="1.0.13" />
     <PackageReference Include="Lykke.Service.Session" Version="1.0.33" />
-    <PackageReference Include="Lykke.SettingsReader" Version="2.3.1" />
     <PackageReference Include="Lykke.SlackNotification.AzureQueue" Version="2.0.4" />
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.2.1" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.2" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Redis" Version="2.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.0.1" />
-    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.2" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="2.4.0" />
```

Comment:
I would prefer to use this version of Swagger. We have already released HFT API with this version.

Reply:
updated